mmdeploy/apis/inference.py

```diff
@@ -16,15 +16,15 @@ def inference_model(model_cfg: Union[str, mmcv.Config],
 
     Examples:
         >>> from mmdeploy.apis import inference_model
-        >>> model_cfg = ('mmdetection/configs/fcos/'
-                         'fcos_r50_caffe_fpn_gn-head_1x_coco.py')
-        >>> deploy_cfg = ('configs/mmdet/detection/'
-                          'detection_onnxruntime_dynamic.py')
+        >>> model_cfg = 'mmdetection/configs/fcos/' \
```

Comment:
Is this line broken by pre-commit automatically? '\' makes API docs not pretty in readthedocs. We used to fix it in PR #495

Reply:
I'll update it. maybe there's a mistake when revert merge commit.